sass/variables/_sizing.scss

```diff
@@ -125,3 +125,12 @@ $base-radio-box-shadow: 0 0 0 1px $grey inset, 0 0 0 9px $white inset, 0 0 0 10p
 $hover-radio-box-shadow: 0 0 0 1px $alert-primary-active inset, 0 0 0 9px $white inset, 0 0 0 16px $alert-primary-active inset;
 $checked-radio-box-shadow: 0 0 0 1px $grey inset, 0 0 0 9px $white inset, 0 0 0 16px $alert-primary inset;
 $active-radio-box-shadow: 0 0 0 8px $alert-primary-active inset, 0 0 0 16px $white inset;
+
+// Tables
+$table-margin: $base-margin;
+$table-row-divider-width: 1px;
+$table-header-padding: $base-margin;
```

Comment:
Aye.. I thought so, too. But if the larger padding is used, the table header is massive. `$base-margin` isn't exact, but it's closer to the mockup than `$base-padding`.. not entire sure what to do here. Maybe tables need their own padding variable for the style guide?